software/ADPL_electron/ADPL_electron.ino

```diff
@@ -39,31 +39,47 @@ Pump pump(PUMP);
 #define KEEP_PUMP_OFF_TIME 1790000   // 30min-10s off time (29 min 50s)
 
 #include "Bucket.h"
-Bucket bucket(BUCKET);
+#define VOLUME 250.0 //300 mL, varies by location
+#define OPTIMAL_FLOW 5.0 //5.0 L/hr, varies by location
+Bucket bucket(BUCKET, VOLUME, OPTIMAL_FLOW);
+
+
+#include "PinchValve.h"
+PinchValve pinchValve(DIR, STEP, SLEEP, UP, DOWN, RESET);
+#define FEEDBACK_RESOLUTION 0.125 // mm of movement 16/turn
+#define PUSH_BUTTON_RESOLUTION 1.0 // mm of movement
+#define HALF_RESOLUTION 0.5 // mm of movement
+#define UNCLOG_RESOLUTION 4.0 // mm of movment
+#define MAX_POSITION 5.0 // in mm
+#define MIN_POSITION 0.0 // in mm
```

Comment:
Are these define constants expected to change for each site, or be something somewhat "fixed" for each pinch valve deployment?  If fixed, then I'm inclined to make these private constants in the `PinchValve` class.